packages/ketcher-react/src/script/render/restruct/index.js

```diff
@@ -592,6 +608,14 @@ ReStruct.prototype.showBonds = function () {
   })
 }
 
+ReStruct.prototype.showText = function () {
+  this.texts.forEach((text, id) => {
+    const path = text.draw(this.render, id)
+    if (path) this.addReObjectPath('data', text.visel, path, null, true)
+    // TODO fragment selection & highlighting
```

Comment:
What do you  need this TODO for?

Reply:
Currently we are treating the whole text as 1 entity, but in #339 we shall be able to add different styles to text parts. So I was thinking, maybe we shall change how text selection works? Give the user an opportunity to select & move text parts separately? Will be happy to discuss.

Reply:
As we plan to add the ability to style text parts separately in #339 , I was thinking, maybe we shall change the text selection in the future? Give the user an ability to open text parts separately? Will be happy to discuss.

Reply:
discussed